README.md

```diff
@@ -533,6 +571,63 @@ project.getMetadata(function(err, metadata) {
 ```
 
 
+## Google Cloud Speech (Limited Preview)
+
+> **This is a Limited Preview release of Google Cloud Speech.** This feature is not covered by any SLA or deprecation policy and may be subject to backward-incompatible changes.
+
+- [API Documentation][gcloud-speech-docs]
+- [Official Documentation][cloud-speech-docs]
+
+```js
+var gcloud = require('gcloud');
+
+// Authorizing on a per-API-basis. You don't need to do this if you auth on a
+// global basis (see Authorization section above).
+
+var speech = gcloud.speech({
+  projectId: 'my-project',
+  keyFilename: '/path/to/keyfile.json'
+});
+
+// Detect the speech in an audio file.
+speech.detect('./audio.raw', {
+  encoding: 'LINEAR16',
+  sampleRate: 16000
```

Comment:
So what's the consensus, should I remove or keep this part? https://github.com/GoogleCloudPlatform/gcloud-node/pull/1407/files#diff-f94b7cae0152dc8625b11579494d6973R894